x/liquidstakeibc/keeper/host_chain_test.go

```diff
@@ -193,17 +225,89 @@ func (suite *IntegrationTestSuite) TestProcessHostChainValidatorUpdates() {
 					UnbondingEpoch:  types.CurrentUnbondingEpoch(hcs[0].UnbondingFactor, epoch),
 					ExchangeRate:    sdk.NewDec(1),
 					DelegatedAmount: sdk.NewInt(100),
+					Delegable:       true,
 				},
 			},
 			validators: []stakingtypes.Validator{
 				{
-					OperatorAddress: "valoper2",
-					Status:          stakingtypes.Bonded,
-					Tokens:          sdk.NewInt(100),
-					DelegatorShares: sdk.NewDec(101),
+					OperatorAddress:     "valoper2",
+					Status:              stakingtypes.Bonded,
+					LiquidShares:        sdk.NewDec(0),
+					ValidatorBondShares: sdk.NewDec(0),
+					Tokens:              sdk.NewInt(100),
+					DelegatorShares:     sdk.NewDec(101),
 				},
 			},
+			expectedDelegableState: map[string]bool{
+				"valoper2": true,
+			},
 			err: fmt.Errorf("error while querying validator valoper2 delegation: decoding bech32 failed: invalid separator index -1"),
+		}, {
+			name: "ValidatorDelegable",
+			hc:   *hcs[0],
+			hcValidators: []*types.Validator{
+				{
+					OperatorAddress: "valoper2",
+					ExchangeRate:    sdk.NewDec(1),
+					Delegable:       false,
+				},
+			},
+			validators: []stakingtypes.Validator{
+				{
+					OperatorAddress:     "valoper2",
+					LiquidShares:        sdk.NewDec(30),
+					ValidatorBondShares: sdk.NewDec(1),
+					Tokens:              sdk.NewInt(100),
+					DelegatorShares:     sdk.NewDec(100),
+				},
+			},
+			expectedDelegableState: map[string]bool{
+				"valoper2": true,
+			},
+		}, {
+			name: "ValidatorNotDelegableCap",
+			hc:   *hcs[0],
+			hcValidators: []*types.Validator{
+				{
+					OperatorAddress: "valoper2",
+					ExchangeRate:    sdk.NewDec(1),
+					Delegable:       true,
+				},
+			},
+			validators: []stakingtypes.Validator{
+				{
+					OperatorAddress:     "valoper2",
+					LiquidShares:        sdk.NewDec(60),
+					ValidatorBondShares: sdk.NewDec(1),
+					Tokens:              sdk.NewInt(100),
+					DelegatorShares:     sdk.NewDec(100),
+				},
+			},
+			expectedDelegableState: map[string]bool{
+				"valoper2": false,
+			},
+		}, {
+			name: "ValidatorNotDelegableBondFactor",
+			hc:   *hcs[0],
+			hcValidators: []*types.Validator{
+				{
+					OperatorAddress: "valoper2",
+					ExchangeRate:    sdk.NewDec(1),
+					Delegable:       true,
+				},
+			},
+			validators: []stakingtypes.Validator{
+				{
+					OperatorAddress:     "valoper2",
+					LiquidShares:        sdk.NewDec(30),
+					ValidatorBondShares: sdk.NewDec(0),
+					Tokens:              sdk.NewInt(100),
+					DelegatorShares:     sdk.NewDec(100),
+				},
+			},
+			expectedDelegableState: map[string]bool{
+				"valoper2": false,
+			},
```

Comment:
missing just a few cases on the extremum values:
when `ValidatorBondShares * LiquidShares == BondFactor` and `LiquidShares / DelegatorShares == LsmValidatorCap`, basically covering those checks where LTE should be LE.